src/components/MoneyRequestHeader.tsx

```diff
@@ -66,18 +45,13 @@ type MoneyRequestHeaderProps = MoneyRequestHeaderOnyxProps & {
     onBackButtonPress: () => void;
 };
 
-function MoneyRequestHeader({
-    session,
-    parentReport,
-    report,
-    parentReportAction,
-    transactionViolations,
-    transaction,
-    shownHoldUseExplanation = false,
-    policy,
-    shouldUseNarrowLayout = false,
-    onBackButtonPress,
-}: MoneyRequestHeaderProps) {
+function MoneyRequestHeader({report, parentReportAction, policy, shouldUseNarrowLayout = false, onBackButtonPress}: MoneyRequestHeaderProps) {
+    const [parentReport] = useOnyx(`${ONYXKEYS.COLLECTION.REPORT}${report.parentReportID}`);
+    const [transaction] = useOnyx(`${ONYXKEYS.COLLECTION.TRANSACTION}${(parentReportAction as ReportAction & OriginalMessageIOU)?.originalMessage?.IOUTransactionID ?? 0}`);
+    const [transactionViolations] = useOnyx(ONYXKEYS.COLLECTION.TRANSACTION_VIOLATIONS);
+    const [session] = useOnyx(ONYXKEYS.SESSION);
+    const [shownHoldUseExplanation] = useOnyx(ONYXKEYS.NVP_HOLD_USE_EXPLAINED, {initWithStoredValues: false});
```

Comment:
@bernhardoj Why do you change the initial value of shownHoldUseExplanation?

Reply:
Oh my bad, I think I got confused between initial value and initWithStoredValues. The initial value for that is false as you can see here,
<img width="275" alt="image" src="https://github.com/Expensify/App/assets/50919443/a7d619b4-c0b6-4790-9ca9-61e75e038469">

but the initWithStoredValues is true.

Here is the PR to correct it: https://github.com/Expensify/App/pull/42819.